### PR TITLE
Prepare the CMS to hold Prevalence Data #119

### PR DESCRIPTION
**Pull request by ckchinmay**

### CODE REVIEW

**Review by dptolle — CHANGES_REQUESTED** [review]
Hi,

sorry I forgot to provide you with english translations for some of the fields. 
Can you please rename the following:
zomoProgramm - zomoProgram
    weitereDetails - furtherDetails
    anzahlProbenN - numberOfSamples
    positiveProbenN - numberOfPositive
    posProbenProz - percentageOfPositive
    konfintervallMin - ciMin
    konfintervallMax - ciMax

File: src/api/prevalence/content-types/prevalence/schema.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "integer"
+    },
+    "zomoProgram": {
+      "type": "string",
+      "private": true
+    },
+    "furtherDetails": {
+      "type": "text"
+    },
+    "numberOfSamples": {
```
Comment (dptolle) [inline src/api/prevalence/content-types/prevalence/schema.json:24]: Should be an integer

File: src/api/prevalence/content-types/prevalence/schema.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "string"
+    },
+    "numberOfPositive": {
```
Comment (dptolle) [inline src/api/prevalence/content-types/prevalence/schema.json:27]: Should be an integer

File: src/api/prevalence/content-types/prevalence/schema.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "string"
+    },
+    "percentageOfPositive": {
```
Comment (dptolle) [inline src/api/prevalence/content-types/prevalence/schema.json:30]: is a percentage

File: src/api/prevalence/content-types/prevalence/schema.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "string"
+    },
+    "ciMin": {
```
Comment (dptolle) [inline src/api/prevalence/content-types/prevalence/schema.json:33]: Should also be a number format

File: src/api/prevalence/content-types/prevalence/schema.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      "type": "string"
+    },
+    "ciMax": {
```
Comment (dptolle) [inline src/api/prevalence/content-types/prevalence/schema.json:36]: Should also be a number format

**Review by dptolle — CHANGES_REQUESTED** [review]

File: src/api/prevalence/content-types/prevalence/schema.json (hunk 2 above)
Comment (ckchinmay) [reply src/api/prevalence/content-types/prevalence/schema.json:27]: I saw this weird value (4,3–49,0) in Positive_Proben_N columns. thus I thought it is better to keep everything string.  I thought I'll check with Carolina if that is valid value or a mistake

**Review by dptolle — APPROVED** [review]

**Pull request merged**